src/test/java/teammates/test/cases/action/InstructorHomePageActionTest.java

```diff
@@ -142,8 +142,8 @@ public void testExecuteAndPostProcess() throws Exception {
         try {
             a = getAction(addUserIdToParams(instructorWithMultipleCourses, submissionParams));
             r = getShowPageResult(a);
-            signalFailureToDetectException("The run time exception is not thrown as expected");
-        } catch (RuntimeException e) {
+            signalFailureToDetectAssertionError("The run time exception is not thrown as expected");
```

Comment:
You need change the message :P

Should not be `The run time exception is not thrown as expected` anymore